lua-mosquitto.c

```diff
@@ -262,6 +262,37 @@ static int ctx_login_set(lua_State *L)
 	return mosq__pstatus(L, rc);
 }
 
+static int ctx_tls_opts_set(lua_State *L)
+{
+	ctx_t *ctx = ctx_check(L, 1);
+	int cert_reqs = luaL_optinteger(L, 2, 0);
+	const char *tls_version	 = luaL_optstring(L, 3, NULL);
+	const char *ciphers = luaL_optstring(L, 4, NULL);
+	
+	// the last param is a callback to a function that asks for a passphrase for a keyfile
+	// our keyfiles should NOT have a passphrase
```

Comment:
These are not the style of comments used in the rest of the file.  Regular C style /* */ please.

Further, if this is important documentation, burying it inside the code isn't the right way of going about it.  
Further, what callback is this?  It's not mentioned anywhere in the mosquitto documentation, nor used here?